### DISCUSSION TITLE
How To Create Texture pack for kubejs?

### DISCUSSION THREAD

**Discussion by Rail3dot14** [NillerMedDild/Enigmatica6#5411] [Q&A]
I was doing texture pack for Create: above and beyond, and ran into a problem - i tried to edit textures of kubejs that was out of the `kubejs.jar` (like kubejs/assets/example), but i didnt know how to. I tried to do it like this:
`assets/kubejs/textures/item/duplicatename.png` 
`assets/kubejs/assets/example/textures/item/dupliactename.png`
`kubejs/assets/example/textures/item/dupliactename.png`
and even replace the textures in kubejs folder

all of that didnt work, so please help

**Comment by MuteTiefling** [comment]
For Create: Above and Beyond? I think you may have the wrong repository. You'd probably get better help asking in the KubeJS Discord as well as I doubt any dev is still happening on Above and Beyond.

**Reply by Rail3dot14** [reply]
thanks, i will try asking on kubejs discord
